incubator/minecraft/templates/deployment.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.Image }}:{{ .Values.ImageTag }}"
+        imagePullPolicy: Always
+        resources:
+          requests:
+            cpu: "{{.Values.Cpu}}"
+            memory: "{{.Values.Memory}}"
+          limits:
+            cpu: "{{.Values.Cpu}}"
+            memory: "{{.Values.Memory}}"
+        env:
+        - name: EULA
+          value: {{ .Values.MinecraftServer.EULA | quote }}
```

Comment:
Holycrap I think you get a cookie for the most ENV vars on this repo. Thank goodness for charts.